### PR TITLE
veux: Added device to OFFICIAL

### PR DESCRIPTION
**Pull request by bexstdev**

### CODE REVIEW

File: veux.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Prateek Bhatia (ꜰʀᴇᴀᴋʏ ᴘʀᴀᴛᴇᴇᴋ)",
+			"oem": "Redmi/Poco",
```
Comment (gwolf2u) [inline veux.json]: Redmi

File: veux.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"device": "veux/peux",
```
Comment (gwolf2u) [inline veux.json]: this should be Note 11 Pro+/Poco X4 Pro

File: veux.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+			"filename": "crDroidAndroid-14.0-20231222-veux-v10.0.zip",
+			"download": "https://sourceforge.net/projects/freaky-builds/files/crDroid/crDroidAndroid-14.0-20231222-veux-v10.0.zip/",
```
Comment (gwolf2u) [inline veux.json]: use generated download url

File: veux.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+			"timestamp": ,
```
Comment (gwolf2u) [inline veux.json]: missing timestamp

File: veux.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+			"md5": "da3fee7e60ab99f8670ff037ee24ca22",
+			"sha256": "2730c5636ba5cd1f853ab790203a98c8e3fab356",
+			"size": 1190239702,
+			"version": "10.0",
+			"buildtype": "Stable",
```
Comment (gwolf2u) [inline veux.json]: Build type: Testing/Alpha/Beta/Weekly/Monthly

File: veux.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+			"forum": "https://xdaforums.com/t/rom-a14-veux-peux-unofficial-crdroid-22-12-23.4647108/",
+			"gapps": "https://sourceforge.net/projects/nikgapps/files/Elite-Releases/crdroid-official/NikGapps-U/",
```
Comment (gwolf2u) [inline veux.json]: try official crDroid Nikgapps
https://nikgapps.com/crdroid-official

File: veux_changelog.txt [hunk 7]
```diff
@@ -0,0 +1,15 @@
+Highlights & Device Specific Changes:
+Build type: Stable
```
Comment (gwolf2u) [inline veux_changelog.txt]: Build type: Testing/Alpha/Beta/Weekly/Monthly

File: veux_changelog.txt [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+Device: veux/peux
```
Comment (gwolf2u) [inline veux_changelog.txt]: Note 11 Pro+/Poco X4 Pro

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Comment by bexstdev** [comment]
Improvements done.

File: veux.json [hunk 9]
```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Prateek Bhatia (ꜰʀᴇᴀᴋʏ ᴘʀᴀᴛᴇᴇᴋ)",
+			"oem": "Redmi",
+			"device": "Redmi Note 11E Pro/Note 11 Pro/Pro+/Poco X4 Pro",
```
Comment (gwolf2u) [inline veux.json]: Keep it to 24 chars length

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: veux.json (hunk 9 above)
Comment (bexstdev) [reply veux.json]: Done

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**